.eslintrc.js

```diff
@@ -1,27 +1,12 @@
+const legacyCode = {
+    'class-methods-use-this': 0,
```

Comment:
There were too many places using this code style for me to add individual "disable" comments. I don’t think having this rule disabled is an issue at all.